azure-pipelines.yml

```diff
@@ -9,10 +9,10 @@ steps:
   - script: python3 -m pip install --upgrade pip setuptools
     displayName: 'Install python tools'
   - script: |
-      make easy-install
-      export PATH=$HOME/.nni/bin:$PATH
+      source install.sh
     displayName: 'Install dependencies'
   - script: |
       cd test/naive
-      PATH=$HOME/.local/nni/node/bin:$PATH python3 run.py
+      export PATH=$HOME/.local/bin:$PATH
```

Comment:
Why we replace $HOME/.local/nni/node/bin with $HOME/.local/bin here?

Reply:
- Makefile will explicitly specify which node(and where) to use in $(BIN_PATH)/nnimanager:
- `echo '$(NODE) main.js $$@' >> $(BIN_PATH)/nnimanager`
- so it's unnecessary to add node/bin in PATH